primefaces/src/main/resources/META-INF/resources/primefaces/tabview/tabview.js

```diff
@@ -139,11 +140,22 @@ PrimeFaces.widget.TabView = PrimeFaces.widget.DeferredWidget.extend({
 
             var $this = this;
 
-            PrimeFaces.utils.registerResizeHandler(this, 'resize.' + this.id + '_align', null, function() {
+            this.resizeHandler = PrimeFaces.utils.registerResizeHandler(this, 'resize.' + this.id + '_align', this.jq, function() {
```

Comment:
i wonder if it makes sense to deregister it automatically on refresh/destroy inside PrimeFaces.utils.registerResizeHandler

Reply:
I was thinking the same thing.  I think for registerScrollHandler and resizeHandler we should.